ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/db-details/advanced-configuration/advanced-configuration.tsx

```diff
@@ -0,0 +1,115 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Messages } from '../../../cluster-overview.messages';
+import OverviewSection from '../../../overview-section';
+import { AdvancedConfigurationOverviewCardProps } from '../../card.types';
+import OverviewSectionRow from '../../../overview-section-row';
+import { DbClusterContext } from 'pages/db-cluster-details/dbCluster.context';
+import { useContext, useState } from 'react';
+import { AdvancedConfigurationEditModal } from './edit-advanced-configuration';
+import { useUpdateDbClusterAdvancedConfiguration } from 'hooks';
+import { AdvancedConfigurationFormType } from 'components/cluster-form/advanced-configuration/advanced-configuration-schema';
+
+export const AdvancedConfiguration = ({
+  loading,
+  externalAccess,
+  parameters,
+}: AdvancedConfigurationOverviewCardProps) => {
+  const {
+    canUpdateDb,
+    dbCluster,
+    queryResult: { refetch },
+  } = useContext(DbClusterContext);
+  const [openEditModal, setOpenEditModal] = useState(false);
+  const [updating, setUpdating] = useState(false);
+  const { mutate: updateDbClusterAdvancedConfiguration } =
+    useUpdateDbClusterAdvancedConfiguration();
+  const handleCloseModal = () => {
+    setOpenEditModal(false);
+  };
+
+  const handleSubmit = async ({
+    externalAccess,
+    sourceRanges,
+    engineParametersEnabled,
+    engineParameters,
+  }: AdvancedConfigurationFormType) => {
+    setUpdating(true);
```

Comment:
can't we just use the loading flag from `useUpdateDbClusterAdvancedConfiguration`?

Reply:
@fabio-silva on success we also wait refetching of dbCluster info to provide last actual data for dbCluster string 63, that's why i used updating as variable🤔

Reply:
ok, I see the point now